stuff/cron/aggregate_feedback.py

```diff
@@ -386,6 +386,86 @@ def aggregate_feedback(dbconn: MySQLdb.connections.Connection) -> None:
                                        global_difficulty_average)
 
 
+def aggregate_reviewers_feedback_for_problem(
+        dbconn: MySQLdb.connections.Connection,
+        problem_id: int) -> None:
+    '''Aggregates the reviewers feedback for a certain problem'''
+    with dbconn.cursor() as cur:
+        cur.execute("""SELECT qn.`contents`
+                       FROM `QualityNominations` as qn
+                       WHERE qn.`nomination` = 'quality_tag'
+                       AND qn.`problem_id` = %s;""",
+                    (problem_id,))
+
+        total_votes = 0
+        seal_positive_votes = 0
+        categories_votes: Dict[Text, int] = {}
```

Comment:
nit:

```suggestion
import collections
from typing import DefaultDict
...
        categories_votes: DefaultDict[str, int] = collections.defaultdict(int)
```

con esto, puedes eliminar L414-L416!